pkg/sql/logictest/testdata/logic_test/cursor

```diff
@@ -42,10 +45,13 @@ FETCH 2 foo
 ----
 
 statement ok
-CLOSE foo
+CLOSE ALL
+
+statement error cursor \"foo\" does not exist
+FETCH 2 foo
 
 statement ok
-COMMIT;
+ROLLBACK;
```

Comment:
small nit: could you add this in a new test case so that we keep the test that ensures that the happy path of close + commit stays working?

Reply:
done